provider/datagen/src/lib.rs

```diff
@@ -156,6 +156,37 @@ pub use icu_provider_blob::export as blob_exporter;
 #[cfg(feature = "fs_exporter")]
 pub use icu_provider_fs::export as fs_exporter;
 
+#[cfg(tbd)]
+mod precomputed {
```

Comment:
Suggestion: move this to its own file named `precomputed_blob.rs` and re-export `ReexportableBlobDataProvider` from lib.rs, with docs explaining how to use it. I don't think we need to feature-gate it.

Reply:
I'm not sure how we should expose it yet. One option would be a `precomputed` feature that allows creating a `ReexportableBlobDataProvider` from a built-in blob containing everything. Another option would be to completely hide the implementation and expose a `impl ExportableProvider`. I'm not super happy that it's a BlobDataProvider but not exposed from the blob crate.

Reply:
I think there's no reason it needs to be a `ReexportableBlobDataProvider`; you can make it `ReexportableBufferProvider` with the impls bound on `P: BufferProvider`.